src/runner.js

```diff
@@ -0,0 +1,51 @@
+const { resolve } = require('path')
+const dpack = require('@etherpacks/dpack')
+const chalk = require('chalk')
+const ethers = require('ethers')
+const ganache = require("ganache")
+const { send } = require('minihat')
+module.exports = runner = {}
+
+runner.run = async (output_dir) => {
+    const options = { logging: { quiet: "true"}}
+    const provider = new ethers.providers.Web3Provider(ganache.provider(options))
+    const signer = provider.getSigner()
+    const multifab_pack = require('../lib/multifab/pack/multifab_full_hardhat.dpack.json')
```

Comment:
`multifab.dpack` , don't emulate "full" convention where it's N/A

Reply:
I see it is in multifab repo
Multifab should only ship `multifab.dpack` with one object and one type

Reply:
Do you only want to cut the 'full_' part or also the network name? Could be useful to avoid hardhat pack files overwriting ethereum pack files

Reply:
just the _full part, keep the rest as was for now